sycl/include/CL/sycl/backend.hpp

```diff
@@ -91,7 +91,7 @@ __SYCL_EXPORT context make_context(pi_native_handle NativeHandle,
                                    const async_handler &Handler,
                                    backend Backend);
 __SYCL_EXPORT queue make_queue(pi_native_handle NativeHandle,
-                               const context &TargetContext,
+                               const context &TargetContext, bool KeepOwnership,
```

Comment:
I think we shouldn't extend this one (shared between different backends) as we only extend level_zero. Also I see there is no "keep-ownership" for make_context just above.

Reply:
Actually, I am fine with this change (after seeing that level_zero::make_queue just redirects to this detail::make_queue)

Reply:
@smaslov-intel is it ok that this change breaks ABI? Shouldn't we keep the old version of this function as well?

Reply:
Yes, let's keep the old version (to be removed at major release 2022.0) and link that to the new one (with keep-ownership = false).

Reply:
I don't see that you keep a version without the new "bool" argument

Reply:
right, adding it.

Reply:
done.